AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -17,16 +17,22 @@ import PackageDescription
 
```

Comment:
Added `AWSSDKCommon` module that has `FileBasedConfiguration`. `FileBasedConfiguration` was previously located under `AWSClientRuntime`, but was moved because identity resolvers in `AWSSDKIdentity` require it AND region providers in `AWSClientRuntime` require it. After all modularization tasks are done, we'll decide what to do with `AWSSDKCommon` & stragglers in `AWSClientRuntime`.